src/node/desktop/test/core/err.test.ts

```diff
@@ -26,16 +26,15 @@ function beUnsuccessful(): Err {
   return new Error('Some error');
 }
 
-describe('Err', () => {
+describe('Err Using Chai Asserts', () => {
   describe('Success helper', () => {
     it('Success return should be falsy', () => {
-      expect(!!beSuccessful()).is.false;
-      expect(beSuccessful() == null);
+      assert.isFalse(!!beSuccessful());
```

Comment:
Wondering if `isFalse` and `isTrue` force a boolean context? Be nice if we don't need the `!!` to force boolean on these. Should be a quick thing to test.

Reply:
isTrue and isFalse expect its arguments to be exactly true or false. There is an .isOk and .isNotOk that accepts truthy/falsey arguments, if that's acceptable

Reply:
For readability of tests maybe we should have our own assert specifically for `Err`, perhaps create `src/node/desktop/test/test-utils.ts` with something like:

```
// Err semantics mean "ok" when falsely, "error" when truthy
function assertSuccess(e: Err) {
  assert.isNotOk(e);
}
function assertError(e: Err) {
  assert.isOk(e);
}
```

Then can just do `assertSuccess(beSuccessfull())`, etc?